chapters/synchronous.tex

```diff
@@ -1043,6 +1044,31 @@ \subsection{Sub-Clock Partitioning}\label{sub-clock-partitioning}
 \end{lstlisting}
 \end{example}
 
+\begin{example}
+Forbidding systems of equations involving different sub-clocks means that the following is forbidden:
+\begin{lstlisting}[language=modelica]
+  Real x=sample(time, Clock(1,100));
+  Real y;
+  Real z=subSample(y, 2)+x;
+equation
+  z=0;
```

Comment:
What is the purpose of including this equation while not having any equation for `y`?  I would find the example more useful if it was complete enough to just try it in a tool.

Reply:
It is complete - and hopefully failing in tools.
I will explain it in more detail